Dockerfile

```diff
@@ -9,9 +9,10 @@ RUN bundle install
 COPY . /app
 EXPOSE 5000
 ENV PORT 5000
-ENV ROUTES_FILE=sp_routes.yml
+#ENV ROUTES_FILE=sp_routes.yml
 ENV UNPACKAGER_URL=http://tng-sdk-package:5099/api/v1/packages
 ENV CATALOGUE_URL=http://tng-cat:4011/catalogues/api/v2
 ENV INTERNAL_CALLBACK_URL=http://tng-gtk-common:5000/packages/on-change
+# To be defined only for the VnV deployment
 ENV EXTERNAL_CALLBACK_URL=http://tng-vnv-lcm:6100/api/v1/packages/on-change
```

Comment:
@jbonnet you can set it empty in the dockerfile as default and we set a value in the ansible scripts for VnV only

Reply:
Ok, @felipevicens